stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        return 1 / (1 + 10 ** ((B - A) / 400))
</pre><p> </p><p>I think this line needs explanation (maybe could be via adding the update equations as mentioned in the previous comment), e.g. this value of 400 hasn't been mentioned  anywhere.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
<p>Done</p>